source/guides/1.1/getting-started.md

```diff
@@ -386,7 +386,7 @@ The generator gives us an entity, a repository, a migration, and accompanying te
 
 ### Migrations To Change Our Database Schema
 
-Let's modify the generated migration to include `title` and `author` fields:
+Let's modify the generated migration (the path to the migration will differ for you, because it contains a timestamp) to include `title` and `author` fields:
```

Comment:
I copied the path to migration and messed up my development database, because it tried to create the books twice 😉